src/defrag.c

```diff
@@ -1064,8 +1064,7 @@ void computeDefragCycles() {
      /* We allow increasing the aggressiveness during a scan, but don't
       * reduce it. */
     if (!server.active_defrag_running ||
-        cpu_pct > server.active_defrag_running)
-    {
+        cpu_pct > server.active_defrag_running) {
```

Comment:
This style is present in many existing codes, so this change is not necessary.

Reply:
Okay :)